packages/Schedular/serverless.yml

```diff
@@ -24,3 +24,5 @@ provider:
 functions:
   schedule:
     handler: src/schedule.schedule
+  clearLogs:
```

Comment:
Bro can we use same scheduler function with params hardcoded in CloudWatch?

Reply:
@VivekFitkariwala  Updated